### ISSUE TITLE
Unable to generate PDF w/ custom-loaded LaTeX renderer

### ISSUE THREAD

**Issue by monkecheese** [pressbooks/pressbooks#927]
### Prerequisites

_Check the boxes below by changing them from `[ ]` to `[x]`._

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with one of our built-in root or book themes?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

When attempting a PDF export, if the selected latex renderer was being loaded from outside of the pressbooks codebase, the export would fail with an error saying that no such latex render file exists.

Ex:
```
Test Warning: require_once(pb4/wp-content/plugins/pressbooks/symbionts/pressbooks-latex/automattic-latex-mathjax.php): failed to open stream: No such file or directory in pb4/wp-content/plugins/pressbooks/symbionts/pressbooks-latex/pb-latex.php on line 140
```

### Steps to Reproduce

1. Use another plugin to load a custom LaTeX render (ex: https://github.com/lumenlearning/candela-utility/blob/master/includes/cu-latex.php#L17-L26)
2. Attempt PDF export using mPDF

**Expected behavior:** PDF to generate successfully

**Actual behavior:** PDF generation fails and error is thrown as described above

#### Root Blog Info

Root Blog ID: 1
Root Blog URL: https://pb4-candela.pantheonsite.io/

#### Browser

Platform: OS X
Browser Name: Chrome
Browser Version: 60.0.3112.113
User Agent String: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36

#### WordPress Configuration

Network URL: https://pb4-candela.pantheonsite.io/
Network Type: Subdirectory
Version: 4.8.1
Language: en_US
WP_ENV: Not set
WP_DEBUG: Disabled
Memory Limit: 512M

#### Pressbooks Configuration

Version: 4.2.0
Root Theme: Pressbooks Publisher
Root Theme Version: 3.1.1

#### Pressbooks Dependencies

Epubcheck: Installed
Kindlegen: Not Installed
xmllint: Installed
PrinceXML: Not Installed
Saxon-HE: Not Installed

#### Must-Use Plugins

hm-autoloader.php: n/a
Pantheon: 0.1

#### Network Active Plugins

Candela Utility: 0.3
Pressbooks: 4.2.0
Pressbooks mPDF: 2.0.0
Snapshot: 2.5.0.1
WP Redis: 0.4.0

#### Root Blog Active Plugins

Pressbooks: 4.2.0

#### Server Configuration

PHP Version: 7.0.23
MySQL Version: 5.5.5
Webserver Info: nginx/1.8.1

#### PHP Configuration

Safe Mode: Disabled
Memory Limit: 512M
Upload Max Size: 100M
Post Max Size: 100M
Upload Max Filesize: 100M
Time Limit: 120
Max Input Vars: 10000
URL-aware fopen: On (1)
Display Errors: On (1)

#### PHP Extensions

cURL: Supported
cURL Version: 7.40.0
imagick: Installed
xsl: Installed

**Comment by monkecheese** [comment]
This commit fixes the issue for me:

https://github.com/pressbooks/pressbooks/pull/928

**Comment by greatislander** [comment]
Fixed in #928. Thanks, @monkecheese!
